pkg/networkmanager/network-interface-members.jsx

```diff
@@ -20,7 +20,7 @@ import cockpit from "cockpit";
 import React, { useState, useContext } from "react";
 import { Button } from "@patternfly/react-core/dist/esm/components/Button/index.js";
 import { Card, CardHeader, CardTitle } from '@patternfly/react-core/dist/esm/components/Card/index.js';
-import { Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core/dist/esm/deprecated/components/Dropdown/index.js'
+import { Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core/dist/esm/deprecated/components/Dropdown/index.js';
 import { Switch } from "@patternfly/react-core/dist/esm/components/Switch/index.js";
```

Comment:
This looks like a rebase error, shouldn't be in the "port to new table implementation" commit